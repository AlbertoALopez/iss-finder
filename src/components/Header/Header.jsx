import React, { Component } from 'react';
import { Menu } from 'semantic-ui-react';
import './_Header.scss';


export default class Header extends Component {
  state = {
    activeItem: 'home',
  }
  render() {
    const { activeItem } = this.state;
    return (
      <Menu
        className="app-header"
        inverted={true}
        borderless={true}
      >
      </Menu>
    );
  }
}
