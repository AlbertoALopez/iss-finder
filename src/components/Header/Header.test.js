import React from 'react';
import renderer from 'react-test-renderer';
import Header from './Header.jsx';

test('Header renders properly', () => {
  const component = renderer.create(
    <Header />
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
