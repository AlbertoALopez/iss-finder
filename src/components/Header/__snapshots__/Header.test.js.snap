exports[`test Header renders properly 1`] = `
<div
  className="ui borderless inverted app-header menu">
  <div
    className="item"
    onClick={[Function]}>
    Home
  </div>
</div>
`;
